lib/bdev/nvme/blockdev_nvme.c

```diff
@@ -390,6 +390,10 @@ probe_cb(void *cb_ctx, const struct spdk_nvme_probe_info *probe_info,
 		return false;
 	}
 
+	if (ctx->controllers_remaining > 0) {
```

Comment:
Actually, I planed to submit another patch to delete controller_remaining and to change condition of for loop from "i < NVME_MAX_CONTROLLERS" to "i < ctx->num_whitelist_controllers" after this patch is merged. May I  put the two modification into this patch?